test/functionalTest/harnessFunctions.sh

```diff
@@ -861,14 +861,22 @@ function orionCurl()
   dMsg Executing the curl-command
   _response=$(eval $command 2> /dev/null)
 
+  if [ ! -f /tmp/httpHeaders.out ]
+  then
+    echo "Broker seems to have died ..."
+  else
+    #
+    # Remove "Connection: Keep-Alive" and "Connection: close" headers
+    #
+    sed '/Connection: Keep-Alive/d' /tmp/httpHeaders.out  > /tmp/httpHeaders2.out
+    sed '/Connection: close/d'      /tmp/httpHeaders2.out > /tmp/httpHeaders.out
+    sed '/Connection: Close/d'      /tmp/httpHeaders.out  
+  fi
 
   #
-  # Remove "Connection: Keep-Alive" and "Connection: close" headers
+  # Unless we remove the HTTP header file, it will remain for the next execution
   #
-  sed '/Connection: Keep-Alive/d' /tmp/httpHeaders.out  > /tmp/httpHeaders2.out
-  sed '/Connection: close/d'      /tmp/httpHeaders2.out > /tmp/httpHeaders.out
-  sed '/Connection: Close/d'      /tmp/httpHeaders.out  
-  
+  \rm -f /tmp/httpHeaders2.out /tmp/httpHeaders.out
```

Comment:
Yes.
If you put a backslash before a unix shell command, then the shell doesn't look for any alias.
I myself always alias 'rm' to 'rm -i' - making rm ask me 'yes/no' for each file I remove.
Here, we don't want any alias to change the way 'rm' works.